core/scripts/go.mod

```diff
@@ -280,7 +280,9 @@ require (
 	github.com/zondax/ledger-go v0.14.1 // indirect
 	go.dedis.ch/fixbuf v1.0.3 // indirect
 	go.etcd.io/bbolt v1.3.7 // indirect
+	go.nhat.io/otelsql v0.12.0 // indirect
```

Comment:
how did you choose this package over other options?
https://github.com/XSAM/otelsql
https://github.com/uptrace/opentelemetry-go-extra/tree/main/otelsqlx